x-pack/legacy/plugins/apm/typings/timeseries.ts

```diff
@@ -14,5 +14,17 @@ export interface RectCoordinate {
   x0: number;
 }
 
+export interface TimeSerie {
```

Comment:
is this spelling of `TimeSerie` intentional or should it be `TimeSeries`?

Reply:
fixed, changed to `TimeSeries`